martin/src/pg/utils.rs

```diff
@@ -12,6 +12,7 @@ use tokio::time::timeout;
 
 use crate::source::UrlQuery;
 
+#[allow(clippy::ref_option)]
```

Comment:
this shouldn't be needed with the newer rust - this is already listed with `expected` two lines below

Reply:
Thx!!  I need to update my rust. By the way, may I know your channel? Stable or nightly? I think it's better to be same with you.

Reply:
always stable :)  I usually update the same day, but I make sure it all passes on the current github CI rust